.github/pull_request_template.md

```diff
@@ -36,3 +36,38 @@ _Please delete options that are not relevant._
 - [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
 - [ ] My changes generate no new warnings
 - [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.
+
+## Project Metadata
+
+`` If there is nothing to fill with, then type: none ``
```

Comment:
I am not sure where you're asking the user to write none, like which section and how?

Reply:
I tried to find a better expression, but can't.. gonna try to think of another one :thinking:

Reply:
shall I add an example? like:

if no requirements.txt then type in ``Requirments`` none => ``Requirments: none``

Reply:
> shall I add an example? like:
> 
> if no requirements.txt then type in `Requirments` none => `Requirments: none`

This sounds good, you can add this